DANS-KNAW/dd-transfer-to-vault
# DD-669 transfer to vault m4 build aip ocfl transfer outbox

Fixes DD-669

# Description of changes
Fixed review feedback M1 - M4
Fixed a bug in InboxWatcher that wouldn't work on network shares on Linux

# How to test
1. `start-preprovisioned-box.py -s dev_archaeology dev_transfer`
2. `deploy.py -t configure-postpublish-workflow -e configure_postpublish_workflow_enabled=true dev_archaeology`
3. `deploy.py -e shared_dd_source_dir='~/Development/dataverse-dans/' -m dd-transfer-to-vault dev_transfer`
4. Create and publish a dataset on dev_archaeology
5. Check the directories on dev_transfer: `/var/opt/dans.knawl.nl/tmp/...` 

# Related PRs
(Add links)
*

# Notify
@DANS-KNAW/dataversedans
